client/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "covidtracker",
```

Doesn't really change anything but just for consistency

```suggestion
  "name": "covidtracker-client",
```